app/Http/Controllers/PostController.php

```diff
@@ -21,42 +21,70 @@ class PostController extends Controller
      * Author: Lampham
      */
     protected $postRepository;
+    protected $answerRepository;
+    protected $tagRepository;
+    protected $userRepository;
+    protected $report;
 
-    public function __construct(PostRepositoryInterface $postRepository)
+    public function __construct(PostRepositoryInterface $postRepository,AnswerRepositoryInterface $answerRepository,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing</br>**Severity**: error</br>**File**: app/Http/Controllers/PostController.php</br>**Line**: 29</br>The first parameter of a multi-line function declaration must be on the line after the opening bracket